static/modules/gamemanager.js

```diff
@@ -269,6 +263,7 @@ class GameManager {
   }
 
   collideWithEntities() {
+    // TODO why is this logic not part of the collision.js file?
```

Comment:
This function acts on the game manager's own `Entity` list. No collision logic is done here really. It could still potentially move, but that was my original thought process.